internal/consts/consts.go

```diff
@@ -0,0 +1,10 @@
+package consts
+
+const (
+	// ProgramName is for displaying help, etc.
+	ProgramName = "kube-linter"
+	// MainURL is where the project info can be found.
+	MainURL = "https://github.com/stackrox/kube-linter"
+	// TemplateURL when formatted with template id, allows user to see help for the given template.
+	TemplateURL = "https://docs.kubelinter.io/#/generated/templates?id=%s"
```

Comment:
Call this `TemplateURLFormat` to make it clear it's not a fully rendered string?

Reply:
Renamed.